packages/prerender-proxy/lib/handlers/prerender-check.ts

```diff
@@ -2,7 +2,7 @@ import "source-map-support/register";
 import { CloudFrontRequest, CloudFrontRequestEvent } from "aws-lambda";
 
 const IS_BOT =
-  /googlebot|Google-InspectionTool|chrome-lighthouse|lighthouse|adsbot\-google|Feedfetcher\-Google|bingbot|yandex|baiduspider|Facebot|facebookexternalhit|twitterbot|rogerbot|linkedinbot|embedly|quora link preview|showyoubot|outbrain|pinterest|slackbot|vkShare|W3C_Validator/i;
```

Comment:
@krishanthisera same with this. Also why is this code duplicated?

Reply:
Again unnecessary escape character. These are two different packages, one is to be used as a library when you want to use the Lambda functions with Middleware.